.github/workflows/status-check.yml

```diff
@@ -1,7 +1,4 @@
-on:
-  pull_request:
-    branches:
-    - main
+on: push
```

Comment:
I'm not sure why this change would make things better - so asking @NSeydoux to review this part.

Reply:
As discussed, it makes sense to only run on PRs to main.